### DISCUSSION TITLE
AMT device switches from ACM to CCM when connected to meshcentral

### DISCUSSION THREAD

**Discussion by Serajalhorany** [Ylianst/MeshCentral#6263] [Issues]
I configured an Intel AMT  device in ACM mode but when I connect it to meshcentral I put the AMT setting to full automatic it and I enter the credintials sometimes it says invalid credintials and sometimes it says in details credintials are known and switches the device to CCM mode.
I tried it on two laptops and still same problem but it works fine with other two laptops. 
My config.json is the default one with the domain name of my ip address. 
I tried amtconfig in console am not an expert but it seems fine.

**Comment by si458** [comment]
sorry ive been out at work.

basically automatic mode will try to activate it into acm mode and if unsuccessful activate into ccm mode?

what is the output from `amtconfig` when u run it?

one other thing to try is go into the console tab in the `my server` page, then type `amtpasswords`, then check the password for the devices amt is actually listed there (this just verifies it actually saved ur password)

**Reply by Serajalhorany** [reply]
Ill check that out right now

**Reply by Serajalhorany** [reply]
I got into the console and promted amtpasswords there are couple of random passwords I dont remember putting and my password

**Reply by si458** [reply]
the random ones are the CCM passwords, as `fully automatic` will generate a password and use it for the intel AMT, so they are ok
so now what is the output from running `amtconfig` on the console tab of a device
and also whats the output of `amtevents`
and also output of `amt`
and can u share a screenshot/snippet of the general tab? (im looking at the intel amt line really)
just building a picture of whats there while thinking

**Reply by Serajalhorany** [reply]
![20240718_212117.jpg](https://github.com/user-attachments/assets/b8af18de-0870-4b96-838d-462367681376)

**Reply by Serajalhorany** [reply]
![20240718_213122.jpg](https://github.com/user-attachments/assets/e1ef26b2-738e-4a83-b8a9-391711be434c)

![20240718_213101.jpg](https://github.com/user-attachments/assets/4f375b57-49ed-4499-901b-470da9994c97)

**Reply by Serajalhorany** [reply]
Am positive that I provisioned the device in ACM mode because I can see user consent in MEBx

**Reply by si458** [reply]
ok and if you go into the Intel AMT tab and then click connect,
what does the `intel me` line say?

**Reply by Serajalhorany** [reply]
Also I provision the device in ACM but when I connect it to meshcentral it goes to CCM and the user consent in MEBx disapears

**Reply by Serajalhorany** [reply]
Also when I look up the intel AMT event logs it says red warnings with to many accessing attemps mybe your getting attacked

**Reply by si458** [reply]
screenshots plz... 
also if you go into the Intel AMT tab and then click connect, what does the intel me line say?

**Reply by Serajalhorany** [reply]
It connects

**Reply by Serajalhorany** [reply]
![Screenshot_20240718_225048_WhatsApp.jpg](https://github.com/user-attachments/assets/fca824e9-3a8e-4c1b-b49e-ea9cc8031486)

**Reply by Serajalhorany** [reply]
There is this screen shot that I I would like you to see

**Reply by si458** [reply]
Right, it's now saying `no trusted dns suffix reported`

You need to go into the intel amt bios (mbex) and set the dns suffix under networking to your dns name of ur meshcentral server

**Reply by si458** [reply]
> Right, it's now saying `no trusted dns suffix reported`
> 
> You need to go into the intel amt bios (mbex) and set the dns suffix under networking to your dns name of ur meshcentral server

You might also be able to connect to the intel amt page via meshcentral then into networking and set the dns suffix that way

**Reply by Serajalhorany** [reply]
![Screenshot_20240719_003821_WhatsApp.jpg](https://github.com/user-attachments/assets/24fa6a0c-0604-4e0c-95a3-148995889f24)

Sorry for the image cant take a screenshot , is this the option your talking about ?

**Reply by si458** [reply]
Yes the domain name! 
It needs to be set to the same dns name u use to access ur meshcentral server 
And also set a computer name too if u fancy

**Reply by Serajalhorany** [reply]
It says set FQDN data error dns is the ip address of my server

**Reply by si458** [reply]
You cannot use ip addresses with intel amt.
You must use a dns name!!!
So ur gunna have to set `cert` in your config.json to a dns name that ur machines can resolve to ur meshcentral ip address
Then set that dns name inside intel amt
Sorry

**Reply by Serajalhorany** [reply]
Ill do that right away. First where I can donate?

**Reply by si458** [reply]
> Ill do that right away. First where I can donate?

I have a few different donation methods on my website :)
https://www.si458.co.uk/2024/01/05/donation/

**Reply by Serajalhorany** [reply]
I routed the ip address with dns name and entered it  and still Set FQDN Data Error

**Reply by si458** [reply]
> I routed the ip address with dns name and entered it  and still Set FQDN Data Error

Screenshot? :)

**Reply by Serajalhorany** [reply]
![Screenshot_20240719_015448_WhatsApp.jpg](https://github.com/user-attachments/assets/cbb81780-86c0-426d-979b-6e32c51e1d26)

Here it is

**Reply by si458** [reply]
You only need to set the suffix to the dns name.

You set the hostname to a unique name for that computer

Example: comp1 and meshcentral.myserver.com

**Reply by Serajalhorany** [reply]
I know am too much trouble 😂 but I hope it works

**Reply by Serajalhorany** [reply]
Still no luck , thanks anyway it seems like I will ask for the code everytime I want to get access

**Reply by si458** [reply]
You will need to run the `amtconfig` again and let it reconfigure amt from ccm to acm if u have ut group still set as fully automatic

**Reply by Serajalhorany** [reply]
Ill try it
